src/firstboot/firstboot.c

```diff
@@ -326,7 +326,7 @@ static int prompt_keymap(void) {
 
 static int process_keymap(void) {
         const char *etc_vconsoleconf;
-        char **keymap;
+        char **keymap = NULL;
```

Comment:
This is unitialized memory access, and I'd prefer to have this as a separate commit with a commit message, because it's something to be backported.

That said, I don't like the fix ;)
In line 355 we know that if `arg_keymap == NULL`, we'll exit in line 359. So instead of doing this in this roundabout way, just exit immediately.

Reply:
Also please consider splitting the rest in two commits: one for the unused variables, and one for the printf annotation.

Reply:
Duh, it was all separate commits, I was looking at the wrong view. I plead lack of coffee. Please ignore my ramblings about splitting into commits.

Reply:
keymap handling was added to firstboot in the current cycle only, hence nothing to backport...

Reply:
So, should I fix the flow of this function? or this fix is acceptble? Now this is labeled as needs-rework... Yeah, actually the flow of the function seems to be weired a little bit...

Reply:
Yeah, please fix the function.

Reply:
Ok. I will try to do. Thanks.